### PR TITLE
Hbar and HbarUnit Parity

### PR DESCRIPTION
**Pull request by danielakhterov**

### CODE REVIEW

File: hbar.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+var HbarMAX = Hbar{max}
+var HbarMIN = Hbar{min}
+var HbarZERO = Hbar{}
+
+func HbarFromTinybar(tinybar uint64) Hbar {
+	return Hbar{tinybar: int64(tinybar)}
+}
+
+func HbarOf(tinybar float64) Hbar {
```
Comment (mehcode) [inline hbar.go]: `tinybar` should not be the param name but `hbar`

File: hbar.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+const max = int64(^uint(0) >> 1)
+const min = -max - 1
+
+var HbarMAX = Hbar{max}
+var HbarMIN = Hbar{min}
+var HbarZERO = Hbar{}
+
+func HbarFromTinybar(tinybar uint64) Hbar {
```
Comment (mehcode) [inline hbar.go]: This should be negative no?

File: hbar.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package hedera
+
+type Hbar struct {
+	tinybar int64
+}
+
+const max = int64(^uint(0) >> 1)
+const min = -max - 1
+
+var HbarMAX = Hbar{max}
```
Comment (mehcode) [inline hbar.go]: This should be `MaxHbar` according to Go convention, `ZeroHbar`, `MinHbar`, etc. 

See: https://golang.org/pkg/math/#pkg-constants

File: hbar_unit.go [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	Tinybar  HbarUnit = "tinybar"
+	Microbar HbarUnit = "microbar"
+	Millibar HbarUnit = "millibar"
+	// Hbar     HbarUnit = "hbar"
+	Kilobar HbarUnit = "kilobar"
+	Megabar HbarUnit = "megabar"
+	Gigabar HbarUnit = "gigabar"
+)
+
+func (unit HbarUnit) GetSymbol() string {
```
Comment (mehcode) [inline hbar_unit.go]: This should be `Symbol`

File: hbar_unit.go [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	case Kilobar:
+		return "kℏ"
+	case Megabar:
+		return "Mℏ"
+	case Gigabar:
+		return "Gℏ"
+	}
+
+	// Unreachable
+	return ""
```
Comment (mehcode) [inline hbar_unit.go]: Panic here

**Comment by danielakhterov** [comment]
Merging becuase @QuestofIranon approved it, but doesn't have write access currently.

**Pull request merged**